npm/vue/test-tsd/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "noEmit": true,
+    "skipLibCheck": true,
```

Comment:
I simplified `compilerOptions` leaving only what is useful.  (E.g. `experimentalDecorators` does not make sense in this context.)